contentcuration/contentcuration/viewsets/base.py

```diff
@@ -75,9 +74,11 @@ def id_value_lookup(self, data):
         else:
             # Could alternatively have coerced the list of values to a string
             # but this seemed more explicit in terms of the intended format.
-            return "[{}]".format(
-                ",".join((str(self.get_value(data, attr)) for attr in id_attr))
-            )
+            id_values = (self.get_value(data, attr) for attr in id_attr)
+
+            # For the combined index, use any related objects' primary key
+            combined_index = (idx.pk if hasattr(idx, 'pk') else idx for idx in id_values)
```

Comment:
Hrm, I'm surprised this wasn't breaking previously - was it stringifying the content node before?

Reply:
Looks like the object being returned there, a `rest_framework.relations.PKOnlyObject`, returns its `pk` from its `__str__` method:

```
ipdb> [(i, str(i)) for i in tuple(id_values)]                                                                                                                             
[(<rest_framework.relations.PKOnlyObject object at 0x7f0a176501d0>, '4ce295a9caf94b28b18376e0452d2507'), ('5361ba5142ba4695b27256e5b74adfdd', '5361ba5142ba4695b27256e5b74adfdd')]
```

So I suppose this could be simplified a bit by just returning `(str(self.get_value(data, attr)) for attr in id_attr)`.